.circleci/test-github-repo.sh

```diff
@@ -23,3 +23,10 @@ terminus build:project:create -n "$SOURCE_COMPOSER_PROJECT" "$TERMINUS_SITE" --t
 terminus site:info "$TERMINUS_SITE"
 # Confirm that the Github project was created
 git clone "$CLONE_URL" "$TARGET_REPO_WORKING_COPY"
+
+# Replace with right docker image.
+cd $TARGET_REPO_WORKING_COPY
+sed -i 's/quay.io\/pantheon\-public\/build\-tools\-ci:6.x/pantheonpublic\/build\-tools\-ci:8.x\-php7.4/g' ./.circleci/config.yml
```

Comment:
Do we need to do this because there are other contexts where the fixture needs to use the 6.x version? If not, why not just update the fixture?

Reply:
hmmm that's a good point. This is for WP-only (iirc); I guess I could just change it there if that makes sense

Reply:
@greg-1-anderson just double checking. This won't be in a fixture, it's the wordpress template project. I still think it should be ok to move it to 8.x-php7.4, just double checking. Please confirm

Reply:
Doing this seems to actually break the behat_tests over there so, with that and having in mind that we have wordpress-composer-managed in the near(ish) future then I'd like to keep it here at least as of now. @greg-1-anderson any objections?